xonsh/history.py

```diff
@@ -505,7 +505,22 @@ def _hist_show(ns=None, hist=None, start_index=None, end_index=None,
 # Interface to History
 #
 class History(object):
-    """Xonsh session history."""
+    """Xonsh session history.
+
+    History is accessed through several sequence-like fields:
+
+    rtns : sequence of ints
```

Comment:
I think that these should be in an Attributes section as per numpydoc https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#class-docstring